charts/validator/templates/deployment.yaml

```diff
@@ -33,12 +33,28 @@ spec:
           hostNetwork: true
           {{- end }}
           image: "{{ .Values.ghost.image.repository }}:{{ .Values.ghost.image.tag | default .Chart.AppVersion }}"
+          {{- if .Values.ghost.commandOverride }}
+          command:
+          {{- range $key, $val := .Values.ghost.commandOverride }}
+            - {{ $val }}
+          {{- end }}
+          {{- end }}
           args:
+          {{- if .Values.ghost.argsOverride }}
```

Comment:
@teghnet this does the same thing, but for arguments, if left out, the default args that we've always had will be used